### PR TITLE
Setting up playbook for phoenix-next

### PR DESCRIPTION
**Pull request by blisteringherb**
#### What's this PR do?
Sets up the Ansible playbook for the phoenix prod server

#### How should this be manually tested?
`ansible-playbook next-production.yml --vault-password-file ~/.ansible_vault_pass.txt`

#### Any background context you want to provide?
* Updated references to php7.1
* Added certs in vault
* Added nginx configuration based on forge/envoyer

#### What are the relevant tickets?
https://github.com/DoSomething/devops/issues/259

### CODE REVIEW

File: roles/dosomething.web/tasks/php-thirdparty.yml [hunk 1]
```diff
@@ -37,5 +37,7 @@
     dest=/etc/php5/fpm/conf.d/20-newrelic.ini
```
Comment (pnasrat) [inline roles/dosomething.web/tasks/php-thirdparty.yml]: Should this be /etc/php/7.1 ?

**Comment by pnasrat** [comment]
This looks great - I think you just missed one place for php 7.1 fix that and we're good!

File: roles/dosomething.web/tasks/php-thirdparty.yml (hunk 1 above)
Comment (blisteringherb) [inline roles/dosomething.web/tasks/php-thirdparty.yml]: NewRelic still uses the package called `php5`, but it supports both php5 and php7.x. https://discuss.newrelic.com/t/repository-for-newrelic-php7/39794/2

**Comment by pnasrat** [comment]
When I setup newrelic on thor I think it wasn't finding the config there -as it's not autoloaded by php7 despite the fact it works with the shared library. Does newrelic monitoring work with this path?

File: roles/dosomething.web/tasks/php-thirdparty.yml (hunk 1 above)
Comment (pnasrat) [inline roles/dosomething.web/tasks/php-thirdparty.yml]: Pretty sure it should be in the new dir structure as php loads it 

```
 sudo grep ^newrelic.license /etc/php/7.1/fpm/conf.d/20-newrelic.ini
newrelic.license = ""
```

**Comment by pnasrat** [comment]
👍  looks good to me though

**Comment by pnasrat** [comment]
please fix the conflicts

**Comment by pnasrat** [comment]
What's blocking this being submitted, as I'd like to make additions on top of this.

**Comment by blisteringherb** [comment]
I kept delaying merging since we kept finding small updates, but I'll go ahead and merge now.

**Pull request merged**